src/main/java/Duke.java

```diff
@@ -64,18 +76,18 @@ public static void main(String[] args) {
         System.out.println(greet);
 
         String line;
-        ArrayList<Task> taskArrayList= new ArrayList<>(); // Array of Tasks
+        ArrayList<Task> taskArrayList = new ArrayList<>(); // Array of Tasks
         int currentNumber = 0; // Current number of tasks
 
```

Comment:
If currentNumber represents the current number of tasks then you can probably rename it to currentTaskNumber for better code readability.